softwareComponents/configuration/bots/umpad.cpp

```diff
@@ -0,0 +1,32 @@
+#include "umpad.hpp"
+
+namespace rofi {
+
+Rofibot buildUMpad( int n ) {
+	return buildUMpad( n, n );
```

Comment:
V celém PR mícháš taby a mezery (v projektu používáme mezery).

Reply:
Ach jo, to není poprvé. Měl jsem na notebooku nějakou zmrvenou konfiguraci VS Code, tak už by to mělo být ok.